tests/sandbox/headers/test_cors.py

```diff
@@ -25,7 +34,17 @@ def test_cors_options(nhsd_apim_proxy_url, method):
 @pytest.mark.parametrize("method", TEST_METHODS)
 def test_cors(nhsd_apim_proxy_url, method):
     """
-    .. py:function:: Test CORS response
+    .. py:function:: Scenario: A user submitting a request with an 'Origin' header receives \
+    an origin header in response
+
```

Comment:
I think its worthwhile adding some more context into this one, something like:

```
CORS allows the API to be used whilst within a web browser, from websites that are hosted on a different domain to that of the API.

This is important as users utilising our API documentation must be able to send test requests from that page using the `try it now` functionality.
```

Reply:
Added a the above above the given/when/then